rstest_macros/src/refident.rs

```diff
@@ -86,3 +86,35 @@ impl MaybeIdent for crate::parse::Attribute {
         }
     }
 }
+
+pub trait MaybePatIdent {
+    fn maybe_patident(&self) -> Option<&syn::PatIdent>;
+}
+
+impl MaybePatIdent for FnArg {
+    fn maybe_patident(&self) -> Option<&syn::PatIdent> {
+        match self {
+            FnArg::Typed(PatType { pat, .. }) => match pat.as_ref() {
+                Pat::Ident(ident) => Some(ident),
+                _ => None,
+            },
+            _ => None,
+        }
+    }
+}
+
+pub trait SetMutability {
+    fn set_mutability(&mut self, mutability: Option<syn::token::Mut>);
+}
+
```

Comment:
Ok, I still prefer to narrow `RemoveMutability` but I'll live with it :smile:

Reply:
Ha, I just missed it in the diff to your branch, let me use this still, I think it's nicer :smile: